modules/client/src/Connext.ts

```diff
@@ -166,6 +167,10 @@ export abstract class ConnextClient extends EventEmitter {
   async redeem(secret: string): Promise<{ purchaseId: string }> {
     return await this.internal.redeemController.redeem(secret)
   }
+
+  async getPaymentHistory(): Promise<PurchasePaymentRow[]> {
```

Comment:
need to add the getPaymentHistoryById to the external class otherwise looks good to me